js/index.js

```diff
@@ -119,6 +122,14 @@ class Game {
     document.querySelector(
       "#current-round-rolls"
     ).textContent = this.rollsInCurrentRound;
+    const scoreboard = document.querySelector("#score-history");
+    scoreboard.textContent = '';
```

Comment:
@perezvon Instead of resetting, what about printing the latest key / value of `Game.scoreHistory`?

Reply:
See #9

Reply:
I will look into that.